hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -93,14 +93,14 @@ public void migrate(@NonNull final MigrationContext ctx) {
                 ? Key.newBuilder().ed25519(bootstrapConfig.genesisPublicKey()).build()
                 : adminKey;
         NodeAddress nodeDetail;
+        final var addressBook = networkInfo.addressBook();
         for (final var nodeInfo : addressBook) {
             final var nodeBuilder = Node.newBuilder()
                     .nodeId(nodeInfo.nodeId())
                     .accountId(nodeInfo.accountId())
-                    .description(nodeInfo.selfName())
-                    .gossipEndpoint(List.of(
-                            endpointFor(nodeInfo.internalHostName(), nodeInfo.internalPort()),
-                            endpointFor(nodeInfo.externalHostName(), nodeInfo.externalPort())))
+                    // Default node description hard coded to the values used currently
+                    .description("node" + (nodeInfo.nodeId() + 1))
```

Comment:
Minor: note that we've introduced an utility `RosterUtils.formatNodeName(long id)` to ensure the consistency and avoid any code duplication. You may want to use it here to make sure that the entire code base always uses the exact same format for node names, everywhere.

Reply:
yes. Changed it. Thanks @anthony-swirldslabs